exercises/simple-linked-list/example.js

```diff
@@ -0,0 +1,72 @@
+function ListElement (value, next) {
+  if (!(this instanceof ListElement))
```

Comment:
Let's use curly braces here, even though they're not strictly needed.  It's the style we've been using.
Same elsewhere-- I won't call out each spot.